### ISSUE TITLE
danish translation

### ISSUE THREAD

**Issue by jonassmedegaard** [CloudCannon/pagefind#500]
Here's a translation into danish (`da`) of the UI JSON data:
```json
{
    "thanks_to": "Jonas Smedegaard <dr@jones.dk>",
    "comments": "",
    "direction": "ltr",
    "strings": {
        "placeholder": "Søg",
        "clear_search": "Nulstil",
        "load_more": "Indlæs flere resultater",
        "search_label": "Søg på dette website",
        "filters_label": "Filtre",
        "zero_results": "Ingen resultater for [SEARCH_TERM]",
        "many_results": "[COUNT] resultater for [SEARCH_TERM]",
        "one_result": "[COUNT] resultat for [SEARCH_TERM]",
        "alt_search": "Ingen resultater for [SEARCH_TERM]. Viser resultater for [DIFFERENT_TERM] i stedet",
        "search_suggestion": "Ingen resultater for [SEARCH_TERM]. Prøv et af disse søgeord i stedet:",
        "searching": "Søger efter [SEARCH_TERM]..."
    }
}
```

**Comment by bglw** [comment]
Thanks! Added in #501 and will go out with the next release 🙂

**Comment by bglw** [comment]
Released in [v1.0.4](https://github.com/CloudCannon/pagefind/releases/tag/v1.0.4) 🎉
